[2024-01-27T15:04:02.710+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing_dag.create_table manual__2024-01-27T15:04:01.326567+00:00 [queued]>
[2024-01-27T15:04:02.722+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing_dag.create_table manual__2024-01-27T15:04:01.326567+00:00 [queued]>
[2024-01-27T15:04:02.724+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-01-27T15:04:02.725+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2024-01-27T15:04:02.727+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-01-27T15:04:02.739+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2024-01-27 15:04:01.326567+00:00
[2024-01-27T15:04:02.745+0000] {standard_task_runner.py:55} INFO - Started process 3429 to run task
[2024-01-27T15:04:02.750+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing_dag', 'create_table', 'manual__2024-01-27T15:04:01.326567+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/user_processing_dag.py', '--cfg-path', '/tmp/tmpqm76_akk']
[2024-01-27T15:04:02.753+0000] {standard_task_runner.py:83} INFO - Job 12: Subtask create_table
[2024-01-27T15:04:02.838+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing_dag.create_table manual__2024-01-27T15:04:01.326567+00:00 [running]> on host 0c5b25b371b7
[2024-01-27T15:04:02.909+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing_dag
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2024-01-27T15:04:01.326567+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-27T15:04:01.326567+00:00
[2024-01-27T15:04:02.924+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2024-01-27T15:04:02.938+0000] {sql.py:315} INFO - Running statement: 
        CREATE TABLE IF NOT EXISTS users(
            firstname TEXT NOT NULL,
            lastname TEXT NOT NULL,
            countryname TEXT NOT NULL,
            username TEXT NOT NULL,
            password TEXT NOT NULL,
            email TEXT NOT NULL

        );
    , parameters: None
[2024-01-27T15:04:02.943+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2024-01-27T15:04:02.960+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing_dag, task_id=create_table, execution_date=20240127T150401, start_date=20240127T150402, end_date=20240127T150402
[2024-01-27T15:04:03.006+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2024-01-27T15:04:03.031+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
